### ISSUE TITLE
Broken regex

### ISSUE THREAD

**Issue by Nao** [elkarte/Elkarte#2831]
Hey there.
I was trying to update my local repo and couldn't because of a commit I hadn't sent.
I don't remember how I was supposed to do that (I made my commit over 2 years ago, lol...), so I'll just point out the error:

https://github.com/elkarte/Elkarte/blob/development/sources/Request.php#L339

(:?(:? ...)
Should be
(?:(?: ...)

Although you could simply use ((, because it's just in a preg_match -- there's no reason to differentiate capturing and non-capturing elements.

PS: I sent a forum with 200K+ posts over to you guys because I think if they can't use W****, I'd rather see them use the next best thing. :-D (And possibly the best thing, since I haven't evaluated Elk in years...)

**Comment by Spuds** [comment]
And here I thought that was some optional start= syntax or something !  I'll probably just change that to (( since as you said nothing is being done with the capture groups even if its properly defined as non capturing.

And thanks for sending someone to give a test drive on Elk !

**Comment by Nao** [comment]
You're welcome. The guy actually hosted Wedge.org for its first 5 years ;) When I had to move, we talked a lot and then discussed moving his forum to Wedge, which failed because he was running SMF 1.1 and Thorsten's open importer only supports SMF 2, IIRC. So I suggested that he at least tries ElkArte because SMF is just too old in my opinion (I had a quick look at SMF 2.1 without installing it, and it still has bugs that Wedge and presumably Elk fixed years ago.)
His forum is at https://forum.mpdb.tv/

Regarding the regex, I'm just eager to help when I can. It's just hard to do pull requests when you don't actually use the software, or even have a test install of it around... I'm really, really impressed by your (and emanuele's) output. How come you both slowed down in early 2015? I stopped around that date because I feature-froze Wedge and nobody was using it so basically I'm only doing the last fixes now that I'm actually using it (yeah, it's quite sad for such a massive volunteer project!). Was that the same for Elk..?

(And here's the mandatory "We could have so much united!" mention.)
